bagit.xml

```diff
@@ -455,7 +455,8 @@ manifest-sha1.txt
               <t>
                 If a <spanx style="emph">filename</spanx> includes a newline
                 (LF), a carriage return (CR),
-                or carriage return plus newline (CRLF) it &must; be
+                carriage return plus newline (CRLF) or
+                percent (%), those characters (and only those) &must; be
```

Comment:
Think we should use “percent sign” rather than “percent”? That'd be in line with the Unicode standard. If we do that we should also replace “newline” with “line feed” for the same reason, which would also make the abbreviation LF more obvious.

Reply:
Concur

Reply:
Done in 75e8889.  Should I also change the other "newline" in the rest of the text to "line feed"? (probably a different PR)